packages/dev/core/src/Debug/debugLayer.ts

```diff
@@ -261,18 +261,34 @@ export class DebugLayer {
         return this._onSelectionChangedObservable;
     }
 
+    /**
+     * the default configuration of the inspector
+     */
+    public config: IInspectorOptions = {

```

Comment:
this would only work per instance of the debugLayer. do you not think this config should be static instead ? It would also benefit the user of Babylon accessing it through the scene.